packages/web-component/src/components/read-along-component/read-along.tsx

```diff
@@ -208,11 +210,12 @@ export class ReadAlongComponent {
    */
   private urlTransform(path: string): string {
     if (
-      this.useAssetsFolder &&
+      this.imageAssetsFolder &&
```

Comment:
Not necessary since `this.imageAssetsFolder` will always exist

Reply:
Is that still going to be true if the html file specifies `images-assets-folder=""`?

Reply:
If we append `/` to the `this.imageAssetsFolder` bit, then `images-assets-folder=""` would effectively become `images-assets-folder="/"`. A user can also do `images-assets-folder=".'` equivalent to `images-assets-folder="./"`.

Reply:
But that would be a bug, no? If it's `""`, we should not add `/` in front.

Reply:
I agree; however, it is an input error. So we need to define how to handle it. If a user provides  `image-assets-folder=""`  it be treated as `"./"`  or `"/"`. I vote `"./"`